cli/src/main/kotlin/commands/ScannerCommand.kt

```diff
@@ -72,6 +72,7 @@ import org.ossreviewtoolkit.scanner.experimental.DefaultPackageProvenanceResolve
 import org.ossreviewtoolkit.scanner.experimental.DefaultProvenanceDownloader
 import org.ossreviewtoolkit.scanner.experimental.DefaultWorkingTreeCache
 import org.ossreviewtoolkit.scanner.experimental.ExperimentalScanner
+import org.ossreviewtoolkit.scanner.experimental.FileBasedPackageProvenanceStorage
```

Comment:
Commit message nit: If you want to stick to American instead of British English, use `behavior` instead of `behaviour`.